net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/users

```diff
@@ -0,0 +1,32 @@
+{% if helpers.exists('OPNsense.freeradius.general.enabled') and OPNsense.freeradius.general.enabled == '1' %}
+
+{% if helpers.exists('OPNsense.freeradius.user.users.user') %}
+{%   for user_list in helpers.toList('OPNsense.freeradius.user.users.user') %}
+{%     if user_list.enabled == '1' %}
+{{ user_list.username }}  Cleartext-Password := "{{ user_list.password }}"
+       Framed-Protocol = PPP,
+{%       if user_list.ip is defined %}
+       Framed-IP-Address = {{ user_list.ip }}{% endif %}{% if user_list.subnet is defined %},
+       Framed-IP-Netmask = {{ user_list.subnet }}{% endif %}{% if user_list.vlan is defined %},
+       Tunnel-Type = VLAN,
+       Tunnel-Medium-Type = IEEE-802,
+       Tunnel-Private-Group-Id = {{ user_list.vlan }}
```

Comment:
This should not be written that way - it should be omitted if not existing (if around the line) or a default value should be used. An empty value can lead to a broken configuration. Also 802.1X has nothing to do with the IP, so the if is misplaced. As a solution I would suggest to end the if before Tunnel-Type and start another one for the VLAN part.